client/src/main/java/org/apache/cassandra/sidecar/client/SidecarClient.java

```diff
@@ -544,6 +551,45 @@ public void streamCdcSegments(SidecarInstance sidecarInstance,
                 .build(), streamConsumer);
     }
 
+    /**
+     * Get configs for all the services in the "configs" table inside sidecar's internal
+     * keyspace
+     *
+     * @return List of services and their corresponding configs
+     */
+    public CompletableFuture<GetServicesConfigPayload> getServicesConfig()
+    {
+        return executor.executeRequestAsync(requestBuilder()
+                       .request(new GetServicesConfigRequest())
+                       .build());
+    }
+
+    /**
+     * Update config for a given service in "configs" table in internal sidecar keyspace
+     *
+     * @param serviceConfig service for which the configs are being updated
+     * @param config the updated config
+     * @return updated config
+     */
+    public CompletableFuture<PutCdcServiceConfigPayload> putCdcServiceConfig(ServiceConfig serviceConfig, Map<String, String> config)
+    {
+        return executor.executeRequestAsync(requestBuilder()
+                       .request(new PutServiceConfigRequest(serviceConfig, new PutCdcServiceConfigPayload(config)))
+                       .build());
+    }
+
+    /**
+     * Deletes configs for a given service in "configs" table in internal sidecar keyspace
+     *
+     * @param serviceConfig service for which the configs are being deleted
+     */
+    public CompletableFuture<Void> deleteCdcServiceConfig(ServiceConfig serviceConfig)
```

Comment:
forgot about this one, is this cdc specific? or is it more a generic config?

Reply:
Its a generic one, but both CDC & KAFKA configs are only used for CDC feature